examples/provider/provider.tf

```diff
@@ -2,7 +2,7 @@ terraform {
   required_providers {
     nobl9 = {
       source  = "nobl9/nobl9"
-      version = "0.13.0"
+      version = "0.16.0"
```

Comment:
please remove `organization` and `project` from the below example, let's keep it minimal:

```terraform
provider "nobl9" {
  client_id     = "<client_id>"
  client_secret = "<client_secret>"
}
```